tests/test_bulk_update.py

```diff
@@ -397,6 +397,31 @@ def test_auto_now(self):
         self.assertEqual(datetime(2020, 1, 2, 0, 0, 0,  tzinfo=tz_utc), instance.checked)
         self.assertEqual(instance.updated, get_auto_now_date())
 
+    def test_auto_now_respects_override(self):
+        res = bulk_update(AutoNowModel, [{
+            'id': 1,
+            'checked': datetime(2020, 1, 2, 0, 0, 0, tzinfo=tz_utc)
+        }])
+
+        instance = AutoNowModel.objects.get()
+        self.assertEqual(datetime(2019, 1, 1,  tzinfo=tz_utc), instance.created)
+        self.assertEqual(datetime(2020, 1, 2, 0, 0, 0,  tzinfo=tz_utc), instance.checked)
+        self.assertEqual(instance.updated, get_auto_now_date())
```

Comment:
This is not required as it dupllicates `test_auto_now`

Reply:
Can remove if you want. This is a style I like to use so that by looking just at the test function you can verify pre and post operation states without relying on other tests being present.

Personally have found this mentality to help as codebases evolve and get refactored and the implicit assumptions and preconditions change.

Can remove if you prefer though.